src/libraries/shims/fakes/Microsoft.Win32.SystemEvents.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <ItemGroup>
+    <TypeForwardedType Include="Microsoft.Win32.PowerModeChangedEventArgs" />
```

Comment:
`TypeForwardType` here is a bit of a misnomer since this doesn't generate any typeforwards, just type stubs.  You might consider a different name.